### ISSUE TITLE
breaking change in date richvalues in 17.x?

### ISSUE THREAD

**Issue by tmtmtmtm** [maxlath/wikibase-cli#184]
v17 seems to have changed the format of richvalue dates over v16:

`wd data --props P569 --simplify --time-converter simple-day --keep qualifiers,ranks,richvalues Q7169784 | jq -r`

under 16:

```json
{
  "id": "Q7169784",
  "type": "item",
  "claims": {
    "P569": [
      {
        "value": {
          "time": "1947-01-18",
          "timezone": 0,
          "before": 0,
          "after": 0,
          "precision": 11,
          "calendarmodel": "http://www.wikidata.org/entity/Q1985727"
        },
        "rank": "normal",
        "qualifiers": {}
      }
    ]
  }
}
```

under 17:

```json
{
  "id": "Q7169784",
  "type": "item",
  "claims": {
    "P569": [
      {
        "time": "1947-01-18",
        "timezone": 0,
        "before": 0,
        "after": 0,
        "precision": 11,
        "calendarmodel": "http://www.wikidata.org/entity/Q1985727",
        "rank": "normal",
        "qualifiers": {}
      }
    ]
  }
}
```

i.e. there is no longer a separate `value` object. I don't see anything in the changelog about this, so I'm not sure if this was expected, or is a regression.

**Comment by maxlath** [comment]
The rank and qualifiers are in the value object, that looks like a bug

**Comment by maxlath** [comment]
That should now be fixed in `v17.0.10`, can you update and retry?

**Comment by tmtmtmtm** [comment]
Yep, seems to be working fine. Thanks for the speedy fix!
